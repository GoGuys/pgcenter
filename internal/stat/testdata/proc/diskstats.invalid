   7       0 loop0 95 0 2452 32 0 0 0 0 0 48
   7       1 loop1 4240 0 9320 1752 0 0 0 0 0 548
   8       0 sda 364890 90257 15905820 98256 5112729 4404633 312756632 3722346 0 3986076
   8      16 sdb 307367 16786 12135182 8494946 84857424 8238986 1047928880 1649683162 0 114626504
