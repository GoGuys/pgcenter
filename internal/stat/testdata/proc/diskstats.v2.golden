   7       0 loop0 95 0 2452 32 0 0 0 0 0 48 4 0 0 0 0
   7       1 loop1 4240 0 9320 1752 0 0 0 0 0 548 552 0 0 0 0
   8       0 sda 365227 90272 15921204 98369 5150316 4436838 318033448 3768000 0 4015784 1972664 391141 0 242608824 1702883
   8      16 sdb 309617 16786 12153678 8579283 85792400 8398482 1060036272 1671783781 0 115793700 1574520404 0 0 0 0
   7       8 loop8 624 0 3532 20 0 0 0 0 0 212 0 0 0 0 0
